package.json

```diff
@@ -55,16 +54,17 @@
     "eslint-plugin-react": "^7.22.0",
     "eslint-plugin-react-native": "^3.10.0",
     "pod-install": "^0.1.14",
-    "react": "17.0.0",
+    "react": "16.13.1",
```

Comment:
Why did you go back to `16.x` ?

Reply:
because React Native v0.63.4 has this version as peer dependency: https://github.com/facebook/react-native/blob/27651720b40cab564a0cbd41be56a02584e0c73a/package.json#L148

Reply:
Hi guys, 

React versions are strongly dependent or the `react-native` version you are using, maybe you should consider having this dependency as a peer dependency (if, ofc, it's not a requirement to get a minimum version) ?

Reply:
Hello @VincentCATILLON. We need it in the `devDependencies` as well for the development purposes, so it doesn't impact production release. Currently we have it in both `devDependencies` and `peerDependencies`.